docs/chapter-3/lesson-3.5.md

```diff
@@ -1,6 +1,6 @@
 # Lesson-3.5
 
-## Library
+## Library (Continued)
```

Comment:
Hey, can you tell me why you added "continued" in parantheses?

Reply:
There was a library lesson in some earlier chapter too where some other library was introduced.. so I thought we had to convey to the readers that this isn't the same lesson. Should I get rid of it 🫤? Or maybe we could do something like:
- Library - Part 1
- Library - Part 2